File: T4/codigos2122/Repaso/app/src/main/java/com/example/repaso/DetailActivity.java
package com.example.repaso;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.ImageView;

import com.example.repaso.utils.Marca;

public class DetailActivity extends AppCompatActivity {

    private ImageView imagen;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_detail);
        imagen = findViewById(R.id.imagen_detalle);
        Marca marcaRecuperada = (Marca) getIntent().getExtras().getSerializable("marca");
        // que imagen pongo aqui????
        imagen.setImageResource(marcaRecuperada.getImagen());

    }
}

File: Kotlin/T3/codigos2223/T3_Spinner_RET/app/src/main/java/com/develop/t3_spinner_ret/DetailActivity.kt
package com.develop.t3_spinner_ret

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.develop.t3_spinner_ret.databinding.ActivityDetailBinding

class DetailActivity : AppCompatActivity() {

    lateinit var binding: ActivityDetailBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_detail)
    }
}